### ISSUE TITLE
The footer links don't have enough contrast 

### ISSUE THREAD

**Issue by joshuadkitenge** [ral-facilities/scigateway#793]
**Description**:

## Dark mode 
Doesn't have enough contrast  to satistfy WCAG 2 AA.Suggested foreground color change to #2685ff .

![footer links(darkmode)](https://user-images.githubusercontent.com/83226114/139670031-8ae4ee7c-b5e7-469a-b921-7aeb64072b2a.png)

**Acceptance criteria**:
- [ ] A checklist
- [ ] of criteria to accept this story as done
